Update README.md (#14)
vtex-apps/availability-notify: docs/README.md

# Availability Notify

> `AvailabilityNotfier` is a VTEX Component that shows the availability notify form that is shown when the product isn't available.

Reviewer: ```suggestion
The Availability Notify component is responsible for showing a subscription form when a product SKU isn't available. The form lets customers subscribe to get notified when that item gets restocked. 

The app records the notification request and monitors inventory updates. This way, once the requested SKU gets back in stock, the app will email the shoppers who asked to be notified.
```

Reviewer: It would be nice to have an image preview of this component implemented in a store.

> `AvailabilityNotfier` is a VTEX Component that shows the availability notify form that is shown when the product isn't available.

Reviewer 2: ```suggestion
`AvailabilityNotifier` is a VTEX Component that provides an availability notification form that is shown when the product being viewed isn't available.
```

> The app records the request for notification and monitors inventory updates.  When the requested sku is back in stock, the app will send an email to the shopper that requested to be notified.

Reviewer: ```suggestion
```

## Configuration

1. [Install](https://developers.vtex.com/vtex-developer-docs/docs/vtex-io-documentation-installing-an-app) the Availability Notify app in the desired VTEX account by running `vtex install vtex.availability-notify` in your terminal.

2. Open your store’s Store Theme app directory in your code editor.

> 3. Add the Availability Notify app to your theme’s in your store theme peer dependencies on `manifest.json`.

Reviewer: ```suggestion
3. Open your app's `manifest.json file` and add the Availability Notify app under the `peerDependencies` field.

```

```json
  "peerDependencies": {
    "vtex.availability-notify": "0.x"
  }
```
4. Add the `availability-notify` component block to your PDP in your store theme (`store.product`). For example:

```json
{
 "store.product": {
    "children": [
      "availability-notify"
    ]
  },

```

> >ℹ️ The email template that will be used is `back-in-stock`

Reviewer: We should specify or link to some other document explaining what is included in this email template and how to change it.

## Configuration

1. [Install](https://developers.vtex.com/vtex-developer-docs/docs/vtex-io-documentation-installing-an-app) the Availability Notify app in the desired VTEX account by running `vtex install vtex.availability-notify` in your terminal.

2. Open your store’s Store Theme app directory in your code editor.

3. Open your app's `manifest.json file` and add the Availability Notify app under the `peerDependencies` field.

```json
  "peerDependencies": {
    "vtex.availability-notify": "0.x"
  }
```
4. Add the `availability-notify` component block to your PDP in your store theme (`store.product`). For example:

```json
{
 "store.product": {
    "children": [
      "availability-notify"
    ]
  },

```

5. Once you have added the `availability-notify` component, access your store's Admin.

7. Go to **Customer** > **Message center** > **Templates**.

9. Search for the `availability-notify` component template, named **BACK IN STOCK** and click on it.

11. After, you will see the email template and its configuration. For example:

![template-back-in-stock](https://user-images.githubusercontent.com/67270558/131547198-a4eb3f0e-5a20-4e63-9f1f-d3bb312fa621.gif)

> >ℹ️ The email template that will be used is `back-in-stock`

Reviewer: ```suggestion
```

```

5. Once you have added the `availability-notify` component, access your store's Admin.

7. Go to **Customer** > **Message center** > **Templates**.

9. Search for the `availability-notify` component template, named **BACK IN STOCK** and click on it.

11. After, you will see the email template and its configuration. For example:

![template-back-in-stock](https://user-images.githubusercontent.com/67270558/131547198-a4eb3f0e-5a20-4e63-9f1f-d3bb312fa621.gif)
>ℹ️ The email template that will be used is `back-in-stock`

Now you set up the template according to your necessities. Check out the next section, [Setting up the Back in stock template](#setting-up-the-back-in-stock-template), to learn how to do it.

> ## Setting up the Back in stock template

Author: @polishq  just added this section to explain the settings of the template. Can you help with this part as well?

Reviewer 2: Hmm, I'm not very experienced with that. Is there any general documentation on editing email templates that we could link to?

Author: There is the [Including order variables in email template](https://help.vtex.com/en/tracks/transactional-emails--6IkJwttMw5T84mlY9RifRP/fLMUCPArCYB9vcTZEZ6bi). I'm going to take a look at it.
